varya/assets/css/style-editor.css

```diff
@@ -121,6 +121,10 @@ a {
 	max-width: var(--responsive--aligndefault-width);
 }
 
+.wp-block[data-align="wide"], .wp-block.alignwide {
+	max-width: var(--responsive--alignwide-width);
```

Comment:
I opted to use this variable instead of `@extend %responsive-alignwide-width;` here because we do not need to apply `margin-left` and `margin-right` rules.